### PR TITLE
Migrate to CCSM Helm chart

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

Migrate to the new [CCSM helm chart](https://github.com/camunda-community-hub/camunda-cloud-helm/tree/main/charts/ccsm-helm), since we will no longer maintain the old charts.

* Update the makefile to use the new charts
* Update the values file to the new structure
* Update the readme

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

### CODE REVIEW

File: benchmarks/setup/default/zeebe-values.yaml [hunk 1]
```diff
@@ -127,7 +150,3 @@ kibana:
 # PROMETHEUS
 prometheus:
   enabled: false
-
-  servicemonitor:
-    enabled: true
```
Comment (npepinpe) [inline benchmarks/setup/default/zeebe-values.yaml:132]: :question: We don't need the service monitor anymore? Does that mean the clusters will not be scraped by our Prometheus install?

**Review by npepinpe — COMMENTED** [review]
:+1: 

Three things:

1. Did you test it? I assume yes, but just want to verify :smile: 
2. The service monitor is not needed anymore?
3. Why did we remove the Operate setup? Are they not using it anymore?

File: benchmarks/setup/default/zeebe-values.yaml [hunk 2]
```diff
@@ -1,23 +1,87 @@
...
+  # IoThreadCount defines how many threads can be used for the exporting on each broker pod
+  ioThreadCount: 4
+
+  # PodSecurityContext defines the security options the broker and gateway container should be run with
+  podSecurityContext:
+    capabilities:
+      add: ["NET_ADMIN"]
+
+  # PrometheusServiceMonitor configuration for the prometheus service monitor
+  prometheusServiceMonitor:
```
Comment (Zelldon) [inline benchmarks/setup/default/zeebe-values.yaml:27]: @npepinpe here it is set

**Comment by Zelldon** [comment]
Hey @npepinpe 

regarding your questions:

1. Yes. I found some nice bugs with it :tada: New released based on that https://github.com/camunda-community-hub/camunda-cloud-helm/releases
2. It is enabled under zeebe
3. They never used it and it is easily possible via the operate.enabled flag.

**Comment by Zelldon** [comment]
http://34.77.165.228/d/I4lo7_EZk/zeebe?orgId=1&refresh=10s&var-DS_PROMETHEUS=Prometheus&var-namespace=zell-test2w&var-pod=All&var-partition=All&from=now-1h&to=now

**Review by npepinpe — APPROVED** [review]
:+1: 

It looks like there were tons of gateways, but now I can't see it since it seems everything is being deleted. Were the metrics just showing many gateways due to restarts?

:thought_balloon: Maybe we need to also have a topology view which includes gateways

**Comment by Zelldon** [comment]
Thanks for the review @npepinpe :bow: 

Yeah I think due to the restarts. Topology view would make sense. BTW I think we can reduce the replica count to 2.

**Pull request merged**